framework/include/controls/Control.h

```diff
@@ -114,11 +114,11 @@ class Control :
   void setControllableValueByName(const std::string & tag, const std::string & param_name, const T & value, bool warn_when_values_differ = false);
   ///@}
 
-private:
```

Comment:
Yes, RELAP-7 has a `connectObject` function in Component that does something similar to `addControllableParameterConnection` and `connectObject` is used in various components, so some way to connect them will be needed that can be used outside of Control.